config/config.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/jinzhu/configor"
+)
+
+//FromJSON will load the requested JSON config into the given config interface
+func FromJSON(config interface{}, name string) error {
+
+	// we need to get the absolute path and also test it because of the
+	// difference of how this code runs in different environments
+	const configPath = "./_config/"
+	absPath, _ := filepath.Abs(configPath + name)
+	_, err := os.Open(absPath)
```

Comment:
Can we use `configor.Load` to implicitly test existence of the file?  Or perhaps `if _, err := os.Stat("/path/to/whatever"); os.IsNotExist(err) {`. If we want to stick with `os.Open` we should use the *File returned and close it (otherwise it is a leaked open file).